roles/validations/tasks/security/invoke_playbooks.yml

```diff
@@ -0,0 +1,10 @@
+- name: Invoke the automation playbook (dataplane certs) on controller-0
+               environment:
```

Comment:
indentation is wrong. `environment` and `cifmw.general.ci_script` must be at same indentation as `name`.